File: mybin/se/sysconfig/formatter/cs/examples/braces_decl.cs
namespace Outer.Namespace {
   class Container {
      string Name {
         get {
            return "Container";
         }
      }
      void statements(boolean cond, int x){
         reg(new ISink() {
            public boolean drain() {
               return false;
            }
         });
      }
   }
}

File: mybin/se/sysconfig/formatter/java/examples/braces_decl.java
class Ladder {
   enum FooState {
      ON, OFF(12) {
         public int[] getSomething() {
            r = new int[] {1, 2, 3};
         }
      }
   }
   void statements(boolean cond, int x) {
      reg(new ISink() {
         public boolean drain() {
            return false;
         }
      });
   }
}
@interface Documented {
   int getRev();
}
